zellij-client/src/os_input_output.rs

```diff
@@ -264,87 +265,6 @@ impl ClientOsApi for ClientOsInputOutput {
-                                if args.bKeyDown == 1 && args.dwControlKeyState == 0 {
-                                    // SAFETY: ASCII is a subset of UTF16 and we called ReadConsoleInputA (the ascii version of the function)
-                                    read_bytes.push(unsafe { args.uChar.AsciiChar });
-                                }
-                                Ok(())
-                            },
-                            windows_sys::Win32::System::Console::MOUSE_EVENT => {
-                                // SAFETY: we just matched the tag
-                                let _args = unsafe { event.Event.MouseEvent };
-                                // TODO implement mouse support. Alacritty does not forward any mouse events in my configuration
-                                // So it is not clear how they arrive here and how they need to be treated.
-                                Ok(())
-                            },
-                            windows_sys::Win32::System::Console::WINDOW_BUFFER_SIZE_EVENT => {
-                                // SAFETY: we just matched the tag
-                                let args = unsafe { event.Event.WindowBufferSizeEvent };
-                                self.send_to_server(ClientToServerMsg::TerminalResize(Size {
-                                    rows: args.dwSize.Y as usize,
-                                    cols: args.dwSize.X as usize,
-                                }));
-                                Ok(())
-                            },
-                            windows_sys::Win32::System::Console::FOCUS_EVENT => {
-                                // Discard as the payload is undocumented: https://learn.microsoft.com/en-us/windows/console/focus-event-record-str
-                                Ok(())
-                            },
-                            windows_sys::Win32::System::Console::MENU_EVENT => {
-                                // Discard as the payload is undocumented: https://learn.microsoft.com/en-us/windows/console/menu-event-record-str
-                                Ok(())
-                            },
-                            x => Err(format!("Unknown Eventtype: {x}")),
-                        };
-                    }
-                    let session_name_after_reading_from_stdin =
-                        { self.session_name.lock().unwrap().clone() };
-                    if session_name_at_calltime.is_some()
-                        && session_name_at_calltime != session_name_after_reading_from_stdin
-                    {
-                        *buffered_bytes = Some(read_bytes);
-                        Err("Session ended")
-                    } else {
-                        Ok(read_bytes)
-                    }
-                } else {
-                    Err(std::io::Error::last_os_error().to_string().leak())
-                }
-            },
```

Comment:
This should have come from `main` as it was the only way I found to enable reacting to size changes of the window.
So I expect that removing this would break that.

Reply:
got it